ios-base/DroidKaigi 2020/FloorMap/FloorMapViewController.swift

```diff
@@ -32,8 +32,8 @@ private extension FloorMapViewController {
     /// This method caches the floor map image in LRU memory when loading from server.
     func loadMap() {
         // Configure cache
-        ImageCache.shared.costLimit = 1024 * 1024 * 5 // 5 MB
-        ImageCache.shared.countLimit = 10
+        ImageCache.shared.costLimit = 1024 * 1024 * 50 // 50 MB
```

Comment:
The problem that operation becomes unstable when the value of `costLimit` and `countLimit ` is as follows was confirmed.

```swift
ImageCache.shared.costLimit = 1024 * 1024 * 5 // 5 MB
ImageCache.shared.countLimit = 10
```

So, I revised the value again.
However, I’m  not sure if this value is appropriate.
I'll find out which value is right for [this part](https://github.com/DroidKaigi/conference-app-2020/pull/775/files#diff-4bb5ebce785a29b29305cf99ad3c3338R35-R36), but I'd be grateful if you could give me an opinion😄